### PR TITLE
Made The Cloud bookable

### PR DESCRIPTION
**Pull request by Supergamer1337**
Cloudlords has shown interest in The Cloud being bookable, so I took the time to make it so. Currently the room has been added and it is protected from being booked privately.

Cloudlords has, as far as I know, not yet created booking conditions for The Cloud yet, so that has not yet been added as a requirement.

### CODE REVIEW

**Review by GAsplund — APPROVED** [review]
Tested and seems to work. LGTM!

File: backend/prisma/schema.prisma [hunk 1]
```diff
@@ -8,23 +8,24 @@ datasource db {
 }
 
 model event {
-  id              String        @id @default(uuid()) @db.Uuid
```
Comment (molleer) [inline backend/prisma/schema.prisma:11]: Please undo the unnecessary reformatting.

**Review by molleer — CHANGES_REQUESTED** [review]
Had one comment, otherwise it looks good

File: backend/prisma/schema.prisma (hunk 1 above)
Comment (Supergamer1337) [reply backend/prisma/schema.prisma:11]: Didn't even notice that the prisma formatter formatted on save... 😅

**Review by molleer — APPROVED** [review]

**Pull request merged**